### PR TITLE
Update ingress-controllers.md

### PR DESCRIPTION
**Pull request by erezyfntn**

### CODE REVIEW

**Comment by erezyfntn** [comment]
Hey.. is it been approved? If yes, when will it be published?

File: content/en/docs/concepts/services-networking/ingress-controllers.md [hunk 1]
```diff
@@ -63,6 +63,8 @@ Kubernetes as a project supports and maintains [AWS](https://github.com/kubernet
 * [Voyager](https://appscode.com/products/voyager) is an ingress controller for
   [HAProxy](https://www.haproxy.org/#desc).
 * [Wallarm Ingress Controller](https://www.wallarm.com/solutions/waf-for-kubernetes) is an Ingress Controller that provides WAAP (WAF) and API Security capabilities.
+* [FortiADC Ingress Controller](https://docs.fortinet.com/document/fortiadc/7.0.0/fortiadc-ingress-controller-1-0/742835/fortiadc-ingress-controller-overview) support the Kubernetes Ingress resources and allows you to manage FortiADC objects from Kubernetes.
```
Comment (windsonsea) [inline content/en/docs/concepts/services-networking/ingress-controllers.md:66]:
```suggestion
* [FortiADC Ingress Controller](https://docs.fortinet.com/document/fortiadc/7.0.0/fortiadc-ingress-controller-1-0/742835/fortiadc-ingress-controller-overview)
  supports the Kubernetes Ingress resources and allows you to manage FortiADC objects from Kubernetes.
```

**Review by windsonsea — COMMENTED** [review]
You need to sign the CLA before review and merge process.

**Review by dipesh-rawat — COMMENTED** [review]
/easycla

File: content/en/docs/concepts/services-networking/ingress-controllers.md (hunk 1 above)
Comment (erezyfntn) [reply content/en/docs/concepts/services-networking/ingress-controllers.md:66]: I already signed the CLA. How can I show it?

**Comment by dipesh-rawat** [comment]
Triggering CLA check again.

/check-cla

**Comment by erezyfntn** [comment]
Done! :)

File: content/en/docs/concepts/services-networking/ingress-controllers.md (hunk 1 above)
Comment (bradtopol) [reply content/en/docs/concepts/services-networking/ingress-controllers.md:66]: Hi @erezyfntn The CLA is good now.

File: content/en/docs/concepts/services-networking/ingress-controllers.md [hunk 2]
```diff
@@ -63,6 +63,9 @@ Kubernetes as a project supports and maintains [AWS](https://github.com/kubernet
 * [Voyager](https://appscode.com/products/voyager) is an ingress controller for
   [HAProxy](https://www.haproxy.org/#desc).
 * [Wallarm Ingress Controller](https://www.wallarm.com/solutions/waf-for-kubernetes) is an Ingress Controller that provides WAAP (WAF) and API Security capabilities.
+* [FortiADC Ingress Controller](https://docs.fortinet.com/document/fortiadc/7.0.0/fortiadc-ingress-controller-1-0/742835/fortiadc-ingress-controller-overview)
+  supports the Kubernetes Ingress resources and allows you to manage FortiADC objects from Kubernetes.
```
Comment (bradtopol) [inline content/en/docs/concepts/services-networking/ingress-controllers.md:67]: The new link and the suggested changes all look good.

**Comment by erezyfntn** [comment]
Hi,

What I'm missing here? what do I need to do to publish it?

**Comment by tengqm** [comment]
> Hi,
> 
> What I'm missing here? what do I need to do to publish it?

Patience.
Wait.

**Review by sftim — COMMENTED** [review]
/lgtm cancel

File: content/en/docs/concepts/services-networking/ingress-controllers.md (hunk 2 above)
Comment (sftim) [reply content/en/docs/concepts/services-networking/ingress-controllers.md:67]: Hi. We keep this list in alphabetical order. Please re-edit to follow the existing sort order.

**Comment by sftim** [comment]
Please fix the ordering - see https://github.com/kubernetes/website/pull/41205#discussion_r1200435028 - and then we can re-review.

A maintainer may close this @erezyfntn if we don't spot an update.

**Comment by mehabhalodiya** [comment]
Hi @erezyfntn, Thanks for raising the PR, please try to follow the template of PR as it will help reviewers to assess your contribution.

**Comment by sftim** [comment]
/close

We would only merge a change that preserved the existing collation.

**Pull request closed without merging**